src/Entity/CommitteeMembership.php

```diff
@@ -98,6 +98,13 @@ class CommitteeMembership
      */
     private $enableVote;
 
+    /**
+     * @var CommitteeCandidacy|null
+     *
+     * @ORM\OneToOne(targetEntity="CommitteeCandidacy")
```

Comment:
what about adding `cascade={"all"}` ?
this way you won't need to bother calling `$manager->persist($committeeCandidacy);`
a simple $manager->flush(); would be enough after assigning the `ComitteeCandidacy` to the `CommitteeMembership`